superset-frontend/cypress-base/cypress/integration/dashboard/markdown.test.ts

```diff
@@ -30,35 +30,44 @@ describe('Dashboard edit markdown', () => {
     cy.get('script').then(nodes => {
       numScripts = nodes.length;
     });
-    cy.get('.dashboard-header [data-test=edit-alt]').click();
```

Comment:
I see there are a lot of these 'edit-alt' ones that got converted to 'pencil' in the changes somehow... maybe an artifact of rebasing?